ESP_Panel_Board_Supported.h

```diff
@@ -69,6 +77,6 @@
  */
 #define ESP_PANEL_BOARD_SUPPORTED_FILE_VERSION_MAJOR 0
 #define ESP_PANEL_BOARD_SUPPORTED_FILE_VERSION_MINOR 1
```

Comment:
```suggestion
#define ESP_PANEL_BOARD_SUPPORTED_FILE_VERSION_MINOR 2
``` 
Sorry. Since it adds a new macro, we should increase the minor version rather than the patch version.

Reply:
done